code/modules/mob/living/bloodcrawl.dm

```diff
@@ -44,32 +59,44 @@
 				sleep(6)
 				if (animation)
 					qdel(animation)
-				playsound(get_turf(src),'sound/misc/Demon_consume.ogg', 100, 1)
-				sleep(30)
-				playsound(get_turf(src),'sound/misc/Demon_consume.ogg', 100, 1)
-				sleep(30)
-				playsound(get_turf(src),'sound/misc/Demon_consume.ogg', 100, 1)
-				sleep(30)
-				src << "<B>You devour [kidnapped]. Your health is fully restored.</B>"
-				src.adjustBruteLoss(-1000)
-				src.adjustFireLoss(-1000)
-				src.adjustOxyLoss(-1000)
-				src.adjustToxLoss(-1000)
-				kidnapped.ghostize()
-				qdel(kidnapped)
-				if (istype(src, /mob/living/simple_animal/slaughter)) //rason, do not want humans to get this
-
-					var/mob/living/simple_animal/slaughter/demon = src
-					demon.devoured++
+				for(var/i = 3; i > 0; i--)
+					playsound(get_turf(src),'sound/misc/Demon_consume.ogg', 100, 1)
+					sleep(30)
+				if (kidnapped)
+					src << "<B>You devour [kidnapped]. Your health is fully restored.</B>"
+					src.adjustBruteLoss(-1000)
+					src.adjustFireLoss(-1000)
+					src.adjustOxyLoss(-1000)
+					src.adjustToxLoss(-1000)
+
+					if (istype(src, /mob/living/simple_animal/slaughter)) //rason, do not want humans to get this
+
+						var/mob/living/simple_animal/slaughter/demon = src
+						demon.devoured++
+						kidnapped << "<span class='userdanger'>You feel teeth sink into your flesh, and the--</span>"
+						kidnapped.adjustBruteLoss(1000)
+						kidnapped.loc = src
```

Comment:
`forceMove()` should always be used over direct setting of loc, if you don't, things like lighting issues will happen.